include/zephyr/drivers/gpio/gpio_utils.h

```diff
@@ -21,10 +21,10 @@
 	((gpio_port_pins_t)(((uint64_t)1 << (ngpios)) - 1U))
 
 #define GPIO_PORT_PIN_MASK_FROM_DT_NODE(node_id)		\
-	GPIO_PORT_PIN_MASK_FROM_NGPIOS(DT_PROP(node_id, ngpios))
+	GPIO_DT_PORT_PIN_MASK_EXC(node_id)
 
 #define GPIO_PORT_PIN_MASK_FROM_DT_INST(inst)			\
-	GPIO_PORT_PIN_MASK_FROM_NGPIOS(DT_INST_PROP(inst, ngpios))
+	GPIO_DT_INST_PORT_PIN_MASK_EXC(inst)
```

Comment:
Why are the aliases needed? Why not just change the implementation of these and leave out the `_EXC` macros?

Reply:
The names "GPIO_PORT_PIN_MASK_FROM_NGPIOS" don't follow the GPIO API naming conventions at all, so IMHO it's  little bit confusing to use 3 different prefixes for the same API: DT_GPIO, GPIO_DT and GPIO_PORT. If for the first part, the reasons was clearly described by Marti Bolivar, the least one is an exception arrived from 2.1.xx epoch. So it's probably better to stop using them and switching the the new macros instead, which following to the common naming scheme and are in the expected location.

However, not touching existing code is also a reasonable decision, and that's exactly what aliasing does.

Reply:
Sorry @galak @henrikbrixandersen for bothering you, but I don't have any response for another week.

Reply:
Hello  @henrikbrixandersen, @nashif, @carlescufi, @galak, @MaureenHelm, @mnkp, @erwango,

Sorry to remind you but this PR has been on ready for over almost a month and on blocking without any response for over one week, is it still useful for Zephyr? And is one response per week a normal code review practice?

Regards,
Andrey VOLKOV